Sources/eppo/ConfigurationStore.swift

```diff
@@ -2,11 +2,20 @@ import Foundation
 
 class ConfigurationStore {
     private var configuration: Configuration?
-    
     private let syncQueue = DispatchQueue(
         label: "com.eppo.configurationStoreQueue", attributes: .concurrent)
     
+    private let fileURL: URL
+    
+    // Initialize with the disk-based path for storage
     public init() {
```

Comment:
What do you think about passing through the `initialConfiguration` optional parameter available from [`EppoClient`](https://github.com/Eppo-exp/eppo-ios-sdk/pull/42/files#diff-dc60401cf877df4122e1cb0b73c011aba7a9a665785dc943c9855d104de4d7e9R69)? We can skip the initial disk read when there's a configuration string.

Reply:
Good idea!